pkg/kds/mux/server.go

```diff
@@ -174,11 +174,11 @@ func (s *server) Start(stop <-chan struct{}) error {
 
 // StreamMessage handle Mux messages for KDS V1. It's not used in KDS V2
 func (s *server) StreamMessage(stream mesh_proto.MultiplexService_StreamMessageServer) error {
-	clientID, err := util.ClientIDFromIncomingCtx(stream.Context())
+	zoneID, err := util.ClientIDFromIncomingCtx(stream.Context())
 	if err != nil {
 		return err
 	}
-	log := muxServerLog.WithValues("client-id", clientID)
+	log := muxServerLog.WithValues("client-id", zoneID)
```

Comment:
Should we use `peer-id` here like in [`pkg/kds/global/components.go`](https://github.com/kumahq/kuma/pull/8897/files#diff-4bfb1199f26a2d0217a66d2f0262151b44248e329a0d7cabd15acb2a1b10e557R120)?

Reply:
I didn't want to change any logs or anything in this PR, perhaps in https://github.com/kumahq/kuma/pull/8898

Reply:
got it